src/components/BookHook.jsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react'
+
+const BookHook = () => {
+  return (
+    <>
+        <div className='flex space-x-[16px] w-[304] ml-[12px] h-[48px] ' >
```

Comment:
you are using tailwind but you are keeping to setting arbitrary values for everything even the ones handled by tailwind